content/learning-paths/cross-platform/kleidiai-explainer/_index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: KleidiAI basics - Improving AI/ML workloads from servers to phones
+
+minutes_to_complete: 60
+
+who_is_this_for: This is an introductory topic for people wanting to learn how Generative AI workloads execute on hardware, and how KleidiAI accelerates it.
+
+learning_objectives: 
+    - Understand how basic math operations power Large Language Models.
+    - Learn how the KleidiAI micro-kernels speed up Generative AI inference performance.
+    - Run a basic C++ matrix multiplication example to showcase the speedup the KleidiAI micro-kernels deliver.
+    
+prerequisites:
+    - An Arm Linux machine that implements the *i8mm* architecture feature; this example uses an AWS Graviton 3 instance.
+    - A basic understanding of linear algebra terminology such as dot product and matrix multiplication.
+
+author_primary: Zach Lasiuk
+### Tags
+skilllevels: Introductory 
+subjects: ML
```

Comment:
+ GenAI, LLM, Coding, Neon?

Reply:
good call, added to 'tools_software_languages'